bundles/FileExplorerBundle/config/services.yaml

```
@@ -0,0 +1,15 @@
+services:
+
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Pimcore\Bundle\FileExplorerBundle\Controller\Admin\:
+        resource: '../src/Controller/Admin'
+        public: true
+        tags: [ 'controller.service_arguments' ]
+    Pimcore\Bundle\FileExplorerBundle\Installer:
```

```suggestion

    Pimcore\Bundle\FileExplorerBundle\Installer:
```

We should add a space here